packages/edit-site/src/components/template-details/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getTemplateInfo } from '../../utils';
+
+export default function TemplateDetails( { template } ) {
+	if ( ! template ) {
+		return null;
+	}
+	const { title, description } = getTemplateInfo( template );
+
+	return (
+		<div className="edit-site-template-details">
+			<p className="edit-site-template-details__heading">
+				{ __( 'Template details' ) }
+			</p>
+			<p>{ `${ __( 'Name' ) }: ${ title }` }</p>
```

Comment:
We should include the punctuation in the translated string (for both Name and Description), as some languages might not use colons.

```diff
- `${ __( 'Name' ) }:`
+ `${ __( 'Name:' ) }`
```

Also, I wonder if we should `sprintf` the title as well. 🤔 
E.g. `sprintf( _x( 'Name: %s', 'template name' ), title )`

Reply:
good idea 👍